src/AdminSettings.vue

```diff
@@ -110,6 +111,25 @@
 				</p>
 			</li>
 		</ul>
+		<ul class="password-generator__settings-list">
+			<li>
+				<CheckboxRadioSwitch :checked.sync="config.genPhrases"
+					type="switch"
+					@update:checked="updateBoolSetting('genPhrases')">
+					{{ t('password_policy', 'Generate passpharses') }}
+				</CheckboxRadioSwitch>
+			</li>
+			<li>
+				<input id="password-gererator__settings__word-count"
```

Comment:
We have a component to input fields. The component already have label. Would be good to use the component [NcInputField](https://nextcloud-vue-components.netlify.app/#/Components/NcFields?id=ncinputfield)

Maybe will be best do this separated in an other PR as a follow up of this pull request because have other places at this vue file with input field and don't will be good mix the purposes of PR.